content/course-specific-material/introR-NMWSC-Mar14.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "New Mexico Water Science Center"
+date: "2017-03-14"
+slug: "introR-NMWSC-Mar14"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+March 14th - 16th in Albuquerque, NM
+
+### Installation
+
+See [Before the Workshop](/intro-curriculum/Before) for information on what software should be installed prior to the course.
+
+### Tentative schedule
+
+**Day 1**
+
+-   08:00 am - 08:30 am -- Instructors available for questions
+-   08:30 am - 10:00 am -- [Introduction](/intro-curriculum/Introduction)
+-   10:00 am - 10:15 am -- *Break*
+-   10:15 am - 12:00 pm -- [Get](/intro-curriculum/Get)
+-   12:00 am - 01:00 pm -- *Lunch*
+-   01:00 pm - 03:00 pm -- [Clean](/intro-curriculum/Clean)
+-   03:00 pm - 03:15 pm -- *Break*
+-   03:15 pm - 04:15 pm -- More [Clean](/intro-curriculum/Clean)
+-   04:15 pm - 04:30 pm -- End of day wrap-up
+
+**Day 2**
+
+-   08:00 am - 08:30 am -- Instructors available for questions
+-   08:30 am - 10:15 am -- [Explore](/intro-curriculum/Explore)
+-   10:15 am - 10:30 am -- *Break*
+-   10:30 am - 12:00 pm -- [Analyze: Base](/intro-curriculum/Analyze)
+-   12:00 pm - 01:00 pm -- *Lunch*
+-   01:00 pm - 02:30 pm -- Visualize with base R: [Visualize](/intro-curriculum/Visualize/)
+-   02:30 pm - 02:45 pm -- *Break*
+-   02:45 pm - 04:15 pm -- Analyze: USGS packages/TBD
+-   04:15 pm - 04:30 pm -- End of day wrap-up
+
+**Day 3**
+
+-   08:00 am - 08:30 am -- Instructors available for questions
+-   08:30 am - 10:00 am -- [Repeat](/intro-curriculum/Reproduce/)
+-   10:00 am - 12:00 pm -- Practice: [USGS R packages](/intro-curriculum/USGS/), projects (group/individual), or [additional topics](/intro-curriculum/Additional/)
```

Comment:
one thing - can you change the end time to 12:30? Then go ahead and merge once it's up